run/types/testing.ts

```diff
@@ -359,13 +361,10 @@ export type Id =
   | 'Image picker'
   | 'network.loki.messenger:id/action_apply'
   | 'Save'
+  | 'Delete'
   | 'android:id/content_preview_text'
-  | 'network.loki.messenger:id/search_result_title';
-  | 'Error message';
+  | 'network.loki.messenger:id/search_result_title'
+  | 'Error message'
   | 'network.loki.messenger:id/action_apply';
 
-export type testRisk = 
-  | 'high' 
-  | 'medium' 
-  | 'low' 
-  | 'undefined';
+export type testRisk = 'high' | 'medium' | 'low' | 'undefined';
```

Comment:
do we really need `'undefined'` and not `undefined` ? this seems like an issue.

Reply:
This was from Miki's PR, can change it to not have quotes

Reply:
yeah we need it without quotes I reckon.

Reply:
done